<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>471</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lookupImgLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>471</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="chooseFileBtn"/>
   <addaction name="separator"/>
   <addaction name="applyLookupTableBtn"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="chooseFileBtn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/dir.png</normaloff>:/images/icons/dir.png</iconset>
   </property>
   <property name="text">
    <string>Choose file</string>
   </property>
   <property name="toolTip">
    <string>Choose file</string>
   </property>
  </action>
  <action name="applyLookupTableBtn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/cleaner.png</normaloff>:/images/icons/cleaner.png</iconset>
   </property>
   <property name="text">
    <string>Apply lookup table</string>
   </property>
   <property name="toolTip">
    <string>Apply lookup table</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
